components/01-atoms/text/headings/headings.yml

```diff
@@ -0,0 +1,42 @@
+-
```

Comment:
Can we make it so that these are on the same line, like:

```
- heading_level: 1
  heading: This is an H1 Example
- heading_level: 2
  heading: This is an H2 Example
- heading_level: 3
  heading: This is an H3 Example
- heading_level: 4
  heading: This is an H4 Example
- heading_level: 5
  heading: This is an H5 Example
- heading_level: 6
  heading: This is an H6 Example
- heading_level: 1
  heading: This is an H1 Example with a link
  heading_url: https://emulsify.info/
- heading_level: 2
  heading: This is an H2 Example with a link
  heading_url: https://emulsify.info/
- heading_level: 3
  heading: This is an H3 Example with a link
  heading_url: https://emulsify.info/
- heading_level: 4
  heading: This is an H4 Example with a link
  heading_url: https://emulsify.info/
- heading_level: 5
  heading: This is an H5 Example with a link
  heading_url: https://emulsify.info/
- heading_level: 6
  heading: This is an H6 Example with a link
  heading_url: https://emulsify.info/
```

Reply:
I've updated this.